Resources/doc/reference/security.rst

```diff
@@ -112,7 +112,7 @@ Let's create the following strategy : a media can be downloaded only once per se
                 return false;
             }
 
-            $this->getSession()->set($this->sessionKey, $times++);
+            $this->getSession()->set($this->sessionKey, ++$times);
```

Comment:
Yes you're right, I just used the ++ to use the same approach but it makes no sense. Changed it to $times + 1